assets/scss/_mixins.scss

```diff
@@ -334,3 +334,22 @@
 @mixin light-text-shadow {
   text-shadow: 0 0 14px var(--color-white);
 }
+
+@mixin hover-scale {
+  $scale-factor: 1.025;
+  $scale-duration: 1.3s;
+  $scale-delay: 0.5s;
+
+  .is-foreground img,
```

Comment:
Vielleicht kann man das mit :where oder :is vereinfachen https://codepen.io/Listor/pen/NWLdQer ?